web/templates/layout/app.html.eex

```diff
@@ -15,6 +16,9 @@
   <body>
     <%= render @view_module, @view_template, assigns %>
 
+
+    </body>
```

Comment:
Looking good. We ended up with 2 closing body and html tags though. Could you remove the new duplicates please?